aclk/README.md

```diff
@@ -13,36 +13,96 @@ The Agent-Cloud link (ACLK) is the mechanism responsible for connecting a Netdat
 [MQTT](https://en.wikipedia.org/wiki/MQTT) over secure websockets to first create, persist, encrypt the connection, and
 then enable the features found in Netdata Cloud. _No data is exchanged with Netdata Cloud until you claim a node._
 
-Read our [claiming documentation](/claim/README.md) for a guide for claiming a node using the ACLK and additional
-troubleshooting and reference information.
+For a guide for claiming a node using the ACLK, plus additional troubleshooting and reference information, read our
+[claiming documentation](/claim/README.md) or the [get started with
+Cloud](https://learn.netdata.cloud/docs/cloud/get-started) guide.
 
 ## Enable and configure the ACLK
 
 The ACLK is enabled by default and automatically configured if the prerequisites installed correctly. You can see this
```

Comment:
No file is created by default, this example does correspond to the default settings but they are stored internally.

Reply:
Got it.